packages/features/data-table/components/DataTableWrapper.tsx

```diff
@@ -79,21 +81,12 @@ export function DataTableWrapper<TData, TValue>({
     }));
   }, [table, sorting, columnFilters, columnVisibility]);
 
+  const showEmptyView = table.getRowCount() === 0 && EmptyView;
+
   return (
-    <DataTable
-      testId={testId}
-      bodyTestId={bodyTestId}
-      table={table}
-      tableContainerRef={tableContainerRef}
-      isPending={isPending}
-      enableColumnResizing={true}
-      hideHeader={hideHeader}
-      variant={variant}
-      className={className}
-      containerClassName={containerClassName}
-      onScroll={(e) => fetchMoreOnBottomReached(e.target as HTMLDivElement)}>
+    <>
```

Comment:
To make it easier to read this diff, it used to be like

```
<DataTable ...>
  <DataTableToolbar.Root>
    some stuff
  </DataTableToolbar.Root>
</DataTable>
```

now it's changed like

```
<div>
  some stuff
</div>
<DataTable ...>
</DataTable>
```